pkgs/native_assets_builder/test_data/drop_dylib_link/hook/link.dart

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:native_assets_cli/native_assets_cli.dart';
+
+void main(List<String> arguments) async {
```

Comment:
Please add a `test/drop_dylib_test.dart` that tries to invoke the function and expect the exception that is thrown because the dylib isn't there. (Also add a test that the other symbol is there.)